### PR TITLE
IDMP-619 add UC3 CQ3

### PR DESCRIPTION
**Pull request by tw-osthus**

### CODE REVIEW

File: etc/unit_tests/configs/idmp_tests_config.yaml [hunk 1]
```diff
@@ -169,4 +169,11 @@ uc3_cq2_tests:
   parameters:
     uc3_cq2_parameter_1: "10020772"
   expected_output: "cq_expected_results/IDMP/uc3_cq2_expected_results.json"  
+
+uc3_cq3_tests:
+  sparql_template: "cq_templates/uc3_cq3.sparql"
+  parameters:
+    uc3_cq2_parameter_1: "100000073619"
```
Comment (mereolog) [inline etc/unit_tests/configs/idmp_tests_config.yaml:176]: uc3_cq2_parameter_1 > uc3_cq3_parameter_1

**Review by avilesg — APPROVED** [review]

**Review by mereolog — CHANGES_REQUESTED** [review]
See https://github.com/edmcouncil/idmp/pull/433#discussion_r1276305679.

**Review by mereolog — CHANGES_REQUESTED** [review]
Some more work on this pull-request is needed because currently uc3_cq3_tests returns an empty result set.

**Comment by tw-osthus** [comment]
It is very strange, because locally the SPARQL runs fine.

**Comment by mereolog** [comment]
> It is very strange, because locally the SPARQL runs fine.

I got the same result, i.e., the empty result set, in [data.world](https://data.world/edm-council/idmp-ontology/workspace/query?queryid=cb8a3518-db03-4c48-92f5-d8267bb12731).

**Comment by tw-osthus** [comment]
Fuseki keeps the old dataset on the disk, even when it has been deleted in the UI. If you re-create it with the same name, it still uses the old version! This was the reason, why I still got results with cmns-pts:isPlayedBy.

**Pull request merged**